lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from 'prop-types'
 
 const Cards = props => {
   return (
     <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+  
+          {props.cards.map((post, i) => {
+            return (
+            <Card 
+            key={i}
+            post={post} 
+            headline={post.headline} 
+            img={post.img} 
+            author={post.author} 
+            />
+          );
+          })}
     </div>
-  )
-}
+  );
+};
 
 // Make sure you include prop types for all of your incoming props
 
+Cards.propTypes = {
+  post: PropTypes.shape({
+    key: PropTypes.string,
+    post: PropTypes.string,
```

Comment:
Because we don't ever have access to the key we don't need to put it in our proptypes, react uses it behind the scenes, but we don't worry about it beyond adding it to the element in the map